src/Symfony/Component/Messenger/Bridge/Redis/Tests/Transport/ConnectionTest.php

```diff
@@ -397,7 +397,7 @@ public static function provideIdPatterns(): \Generator
     {
         yield 'No delay' => ['/^THE_MESSAGE_ID$/', 0, 'xadd', 'THE_MESSAGE_ID'];
 
-        yield '100ms delay' => ['/^[a-f\d]+$/', 100, 'rawCommand', '1'];
+        yield '100ms delay' => ['/^[A-Z\d\/+]+={0,2}$/i', 100, 'rawCommand', '1'];
```

Comment:
do we need to check for the the `=` at the end? 9 bytes never generates a `=`, does it?

Reply:
it's indeed not needed